### DISCUSSION TITLE
fallback_module bug?

### DISCUSSION THREAD

**Discussion by mentlergy** [tasmota/decode-config#86] [General]
Hello!

I have set the fallback_module (module2 in the console) to 0. If I get the configuration with the decode-config I get 255 in the fallback_module variable and if generate script it contains module2 255 statement.
With the tasmotas containing module2 1 the fallback_module variable contains 0.
It seems to be the variable contains the right value minus one.
My tasmota version is 12.1.1

Can you confirm?

**Comment by curzon01** [comment]
it's a bug for `-T cmnd` output only, `fallback_module` value is correct - see #87
